### ISSUE TITLE
Crash when preferred-install is a hash

### ISSUE THREAD

**Issue by notFloran** [eloquent/composer-config-reader#12]
When I use the component with the composer.json of a Symfony Flex project I got the following error : 

```
  [Eloquent\Enumeration\Exception\UndefinedMemberException]
  No member with value equal to stdClass::__set_state(array(
     '*' => 'dist',
  )) defined in class 'Eloquent\\Composer\\Configuration\\Element\\Installati
  onMethod'.
``` 

The problem seems to come from this part of the composer.json file:

```
    "config": {
        "preferred-install": {
            "*": "dist"
        }
    },
```

**Comment by ezzatron** [comment]
Fixed in [2.1.0](https://github.com/eloquent/composer-config-reader/releases/2.1.0).

**Comment by notFloran** [comment]
@ezzatron Thank you for your fast answer !
